app/src/main/java/com/cornellappdev/android/eatery/network/NetworkUtilities.java

```diff
@@ -97,4 +106,26 @@ public void onFailure(@NotNull ApolloException e) {
       }
     });
   }
+
+  public static void getCtEateries(Activity activity) {
+    buildApolloClient();
+
+    final AllCtEateriesQuery ctEateriesQuery = AllCtEateriesQuery.builder().build();
+    ApolloCall<AllCtEateriesQuery.Data> ctEateryCall = apolloClient.query(ctEateriesQuery);
+    ctEateryCall.enqueue(new ApolloCall.Callback<AllCtEateriesQuery.Data>() {
+      @Override
+      public void onResponse(@NotNull Response<AllCtEateriesQuery.Data> response) {
+        collegetownEateriesLoaded = true;
+        collegetownEateries = response.data().collegetownEateries();
+        ArrayList<EateryBaseModel> collegetownEateryList = JsonUtilities.parseCtEateries(activity, collegetownEateries);
+        Collections.sort(collegetownEateryList);
+        rInstance.setCtEateryList(collegetownEateryList);
+      }
+
+      @Override
+      public void onFailure(@NotNull ApolloException e) {
+        collegetownEateriesLoaded = false;
```

Comment:
you don't use this variable. what is the intended action if ct eateries do fail to load? are we hoping to have a static backup or should we disable the toggle pill?

Reply:
I added in the ui branch(next one) to disable the pill.

Reply:
oh nice